### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by meeraharidasa**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -13,113 +13,510 @@
...
+sdk/tables/           @klaaslanghout
+
+# ServiceLabel: %TimeseriesInsights %Service Attention
+sdk/timeseriesinsights/           @Shipra1Mishra
+
+# ServiceLabel: %vFXT %Service Attention
+#/<NotInRepo>/            @zhusijia26
+
+# ServiceLabel: %Web Apps %Service Attention
+#/<NotInRepo>/            @AzureAppServiceCLI @antcp
```
Comment (lmazuel) [inline .github/CODEOWNERS:522]: sdk/appservice/azure-mgmt-web  ?

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -13,113 +13,510 @@
...
 
-# PRLabel: %Consumption
-/sdk/consumption/                                                    @sandeepnl
+# ServiceLabel: %ARM - Core %Service Attention
+#/<NotInRepo>/            @armleads-azure
 
-# PRLabel: %Container Instances
-/sdk/containerinstance/                                              @samkreter @xizhamsft
+# ServiceLabel: %ARM - Managed Applications %Service Attention
+#/<NotInRepo>/            @armleads-azure
```
Comment (lmazuel) [inline .github/CODEOWNERS:51]: sdk/resources/azure-mgmt-resource/azure/mgmt/resource/managedapplications

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -13,113 +13,510 @@
...
-# PRLabel: %Container Registry
-/sdk/containerregistry/                                              @djyou
-/sdk/containerservice/                                               @samkreter @zqingqing1 @GaneshaThirumurthi
+# ServiceLabel: %ARM - Service Catalog %Service Attention
+#/<NotInRepo>/            @armleads-azure
 
-# PRLabel: %Cosmos
-/sdk/cosmos/                                                         @southpolesteve @zfoster
+# ServiceLabel: %ARM - RBAC %Service Attention
+#/<NotInRepo>/            @armleads-azure
```
Comment (lmazuel) [inline .github/CODEOWNERS:57]: sdk/authorization/azure-mgmt-authorization

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -13,113 +13,510 @@
...
 
-# PRLabel: %Cognitive - Text Analytics
-/sdk/textanalytics/                                                  @kristapratico @iscai-msft
+# ServiceLabel: %Batch %Service Attention
+sdk/batch/          @mksuni @bgklein @mscurrell
 
-# PRLabel: %Cognitive - Form Recognizer
-/sdk/formrecognizer/                                                 @kristapratico @iscai-msft @rakshith91
+# ServiceLabel: %BatchAI %Service Attention
+#/<NotInRepo>/            @matthchr
```
Comment (lmazuel) [inline .github/CODEOWNERS:99]: batchai is a deprecated service, I thinkg it's no longer in prod

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -13,113 +13,510 @@
...
+sdk/cognitiveservices/azure-cognitiveservices-personalizer/          @dwaijam
+
+# ServiceLabel: %Cognitive - Immersive Reader %Service Attention
+#/<NotInRepo>/            @metanMSFT
+
+# ServiceLabel: %Cognitive - Ink Recognizer %Service Attention
+#/<NotInRepo>/            @olduroja
+
+# ServiceLabel: %Cognitive - Bing %Service Attention
+#/<NotInRepo>/            @jaggerbodas-ms @arwong
```
Comment (lmazuel) [inline .github/CODEOWNERS:156]: sdk/cognitiveservices/azure-cognitiveservices-search-*

File: .github/CODEOWNERS [hunk 6]
```diff
@@ -13,113 +13,510 @@
...
+# ServiceLabel: %ARM %Service Attention
+#/<NotInRepo>/            @armleads-azure
 
-# PRLabel: %Monitor - LogAnalytics
-/sdk/loganalytics/                                                   @alexeldeib
+# ServiceLabel: %ARM - Templates %Service Attention
+#/<NotInRepo>/            @armleads-azure
 
-# PRLabel: %Monitor - Exporter
-/sdk/monitor/opentelemetry-exporter-azuremonitor                     @rakshith91 @lmazuel @hectorhdzg @lzchen
```
Comment (hectorhdzg) [inline .github/CODEOWNERS:48]: We still need owners for this project, any reason why this is being removed?

**Pull request closed without merging**